src/main/java/org/usfirst/frc/team1294/robot/commands/ClimbRope.java

```diff
@@ -0,0 +1,44 @@
+package org.usfirst.frc.team1294.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import org.usfirst.frc.team1294.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class ClimbRope extends Command {
+
+  public ClimbRope() {
+    super("Drive motor");
+    requires(Robot.climbingSubsystem);
+  }
+
+  @Override
+  protected void initialize() {
+
+  }
+
+  @Override
+  protected void execute() {
+    XboxController joystick = Robot.oi.getJoystick();
```

Comment:
this code should not be here.... it belongs in OI.... all that should be here is to control the talons.